packages/rs-drive-abci/.env.testnet

```
@@ -21,12 +21,12 @@ DATA_CONTRACTS_BLOCK_CACHE_SIZE=200
 # DashCore JSON-RPC host, port and credentials
 # Read more: https://dashcore.readme.io/docs/core-api-ref-remote-procedure-calls
 CORE_CONSENSUS_JSON_RPC_HOST=127.0.0.1
-CORE_CONSENSUS_JSON_RPC_PORT=9998
+CORE_CONSENSUS_JSON_RPC_PORT=19998
 CORE_CONSENSUS_JSON_RPC_USERNAME=dashrpc
 CORE_CONSENSUS_JSON_RPC_PASSWORD=password
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The CORE_CONSENSUS_JSON_RPC_PASSWORD key should go before the CORE_CONSENSUS_JSON_RPC_PORT key

(UnorderedKey)